src/dataset/custom_dataset.py

```diff
@@ -421,7 +424,7 @@ def sample_episode_indices(self, num_eps, dist="uniform"):
             probs = 1 / self.episode_lengths
         else:
             probs = np.ones(self.num_episodes)
-        probs /= np.sum(probs)
+        probs = probs / np.sum(probs)
```

Comment:
this seems to break when using the /= notation with "length" (due to one being float, the other int)